<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ARS-MSOFT-LAP" name="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01" tests="9" failures="0" timestamp="4 Mar 2021 12:02:06 GMT" time="0.014" errors="9">
  <testcase name="verifyInsufficiencyFulfillment" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyInsufficiencyFulfillment()[pri:3, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyInsufficiencyFulfillment()[pri:3, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyInsufficiencyFulfillment -->
  <testcase name="verifyInsufficientVerifier" time="0.014" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="org.testng.TestNGException" message="
Parameter &#039;browser&#039; is required by @Configuration on method setup but has not been marked @Optional or defined
in /tmp/testng-eclipse-1528404305/testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'browser' is required by @Configuration on method setup but has not been marked @Optional or defined
in /tmp/testng-eclipse-1528404305/testng-customsuite.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:154)
	at org.testng.internal.Parameters.createParameters(Parameters.java:361)
	at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:84)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:201)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyInsufficientVerifier -->
  <testcase name="verifyInsufficientCasesPresentOrNot" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyInsufficientCasesPresentOrNot()[pri:4, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyInsufficientCasesPresentOrNot()[pri:4, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyInsufficientCasesPresentOrNot -->
  <testcase name="verifyAdminUserReasonInsufficientBoth" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyAdminUserReasonInsufficientBoth()[pri:7, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyAdminUserReasonInsufficientBoth()[pri:7, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyAdminUserReasonInsufficientBoth -->
  <testcase name="verifyApprovalRequiredInsufficient" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyApprovalRequiredInsufficient()[pri:6, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyApprovalRequiredInsufficient()[pri:6, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyApprovalRequiredInsufficient -->
  <testcase name="verifyApprovalRequiredAndInsufficientBoth" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyApprovalRequiredAndInsufficientBoth()[pri:8, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyApprovalRequiredAndInsufficientBoth()[pri:8, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyApprovalRequiredAndInsufficientBoth -->
  <testcase name="verifyCostApprovalRequiredInsufficient" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyCostApprovalRequiredInsufficient()[pri:5, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyCostApprovalRequiredInsufficient()[pri:5, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyCostApprovalRequiredInsufficient -->
  <testcase name="verifywithdrawCaseStatusClosedByClient" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifywithdrawCaseStatusClosedByClient()[pri:9, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifywithdrawCaseStatusClosedByClient()[pri:9, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifywithdrawCaseStatusClosedByClient -->
  <testcase name="verifyInsufficientpendingForFulfilment" time="0.000" classname="com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01">
    <error type="java.lang.Throwable" message="Method TC_InsufficientBridgeSme_01.verifyInsufficientpendingForFulfilment()[pri:2, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TC_InsufficientBridgeSme_01.verifyInsufficientpendingForFulfilment()[pri:2, instance:com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01@652ce654] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- verifyInsufficientpendingForFulfilment -->
</testsuite> <!-- com.iBridge.sme.testcases.TC_InsufficientBridgeSme_01 -->
